packages/block-library/src/navigation/edit/index.js

```diff
@@ -676,6 +716,7 @@ function Navigation( {
 						/>
 					) }
 					{ ! hasResolvedCanUserCreateNavigation ||
+						isResolvingClassicMenuConversion ||
 						( ! isEntityAvailable && ! isPlaceholderShown && (
 							<PlaceholderPreview isLoading />
```

Comment:
We need to look again at loading states because I don't yet see this. Perhaps because of the call to `await createNavigationMenu` (which again has no resolution status indicators).